NIPostsReload/Presentation/Custom/CustomTabView/Cell/CustomTabViewCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CustomTabViewCell.swift
+//  CustomTabView
+//
+//  Created by Denys Niestierov on 16.11.2023.
+//
+
+import UIKit
+
+final class CustomTabViewCell: UICollectionViewCell {
+    private struct Constant {
+        static let titleFontSize: CGFloat = 18
+        static let titleNumberOfLines = 1
```

Comment:
it's a default value for label
you don't need to set it up, let alone put it to constants